pkg/virt-controller/watch/drain/disruptionbudget/disruptionbudget.go

```diff
@@ -69,9 +75,31 @@ func NewDisruptionBudgetController(
 		UpdateFunc: c.updatePodDisruptionBudget,
 	})
 
+	c.podInformer.AddEventHandler(cache.ResourceEventHandlerFuncs{
+		UpdateFunc: c.updatePod,
+	})
+
 	return c
 }
 
+func (c *DisruptionBudgetController) updatePod(_, curr interface{}) {
+	pod := curr.(*corev1.Pod)
+
+	if pod.DeletionTimestamp != nil {
+		return
+	}
+
+	controllerRef := v1.GetControllerOf(pod)
+	if controllerRef == nil {
+		return
+	}
+	vmi := c.resolveControllerRef(pod.Namespace, controllerRef)
+	if vmi == nil {
+		return
+	}
+	c.enqueueVirtualMachine(vmi)
+}
+
```

Comment:
if you add an updateMigration function similar to this that enques the VMI when the migration changes, then it's safe to remove that check I asked for where we introspect vmi.Status.MigrationState.Completed when determining if the migratino is complete. 

i should have suggested that to begin with, i just didn't connect that when you transitioned to looking at a migration informer that you'd be able to queue the VMI. sorry :/

Reply:
sure, makes sense!